spec/resources/defaultStyles/control-image

```diff
@@ -1 +1 @@
5XscPaYsoSHq2e3lFBX/+x92auzEqfrwndftcnnmuLx0np/e6NnF/j5umbtlC6YqZ/bLS/wldd2VDR48dFiP1G+hZfOnqP/Awdq09Xt7igl8xo0cYpfOMzPHsmW9vBeTadfb20dvvd7N/t7sq9TzjQFqULeWKlUoZ/9s+46devqFy0v62Wu6vSJfH299Nm5yfNhm/tws9deh/dMKCgxU+NFjdgaYOQiaUvRYcBICCCCAAAIIIIAAAggggAACCCCAAAIIIIAAAp4tkFzQZPZDMrNczGyc2fOXxIc8CWcMxclt3LJN7Tt2lwlYvpo+55YsnRcXgnR/9SUdP3FSJiiaM/XyHkoJg6bFS1ck27+EQdP8xcs0Ztzk+HvcSNW/++FHzZi1QP16d4kPmcz1X341Q2u/3aTPhn1gb2f2kXqt91tat3yu3cMquetM+FTqvmLxXTABT9VHm9plAn19fXUh8oL9ma+3j3LkCLOzpIYPGaiSJS5f0/KZDjYwM8v+mSCw3zuDVL9ODZUvWzr+npGRF3U4PDz+91lCgvX3/oN68pkO2rR6kV2mz8yCKl+ljqZP/FwfDRulHTt/tv02x7nz5+3eW2VKldCooe/fCNdV53qdOXsu9l/dgYsRQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEDALQWSC5o6d++nwe/109gvp2jTlu80cuj7NtSo+mgTfTpogDJlyqjf/9ir6lUryyz/9lCtRpr91ReaNnOedv32h4a83z/J8T5Yrb4+GNBXFR8op59+2a02z3VKco8m06/W7V7W8gVTdPT4CTVo2kaTx41Q0XuKJAqaJk+blWT/zIyrp9q/qhpVK6tl88Z2/6E6jVtqwuefqNi9dyXq28rV67Rxy3fqlcTspq3f/WBnMpl9mEzwYw8vL+XKmV2Hjxy1+yctmvWlXTrw/Y+GWwtz7rWuGzR0lJ5p87iCM2eytzN7TE2dMUczJ4+JD3oSdtDsyRQcnFkvPNtGx4+fVL2mrTR32jiFBGdWj74DVK1KRZUsXjT+kkyZguL3c0p4H9O3Ji3bqXe3l1Wm1H1a9c16fTpqrN2z6cTJUzLhVNwxcNBQZcuSxc5UCwkJ/lfPLkHTv+LjYgQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEE3FPgr3377VJ402fNV7WHKykoY6BiY2J1JiJCe/fus3si/b3/gJ57qZsNlkqXLKG16zfb2TgmgHi1xxu6q3BBXTgfqbuKFNJTrVrox50/68VXe+q+Eveqc8fnVTB/vkSD/3zsRJm9gooXvVsFC+TX5Kmz9GbvrmpYv7bKPVRHtWo8rAD/DPrr7316ps0TKlempL1++qwFGjdpmnKEZVPOnNntTKvXXn1RVatUTLJ//ft01eRpszVyzAQ9VPEBvdGzs1au+VZDh49R5uBMypcntw2gihe7xy6pt/DrFZo77fKMqYSHWUJuwdfLE/2Zj7e3fti0wv6ZCanmLFyi0JDMirxwUX17vip/f3+79Fxy15m+z1mwRPnz5bF25yMvqNvLLyhPnlxJPihmdlH/AYMVEBigQ4ePqHnjBqpSuYJM/eo0evKqazq2f1rPP9MqyXv9uecvDfpkpHJmz679Bw6qe5cOyndHniTHHZY9mzp3eO5fP7wETf+akBsggAACCCCAAAIIIIAAAggggAACCCCAAAIIIIDA9QRM0DR1woirwqnrXcfP3VuAoMm960PvEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBwhIAJmiaP/VSFCxVwxHgYxGUBgiaeBAQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEELitAktXrNFrfd5WqxZN1OqJxxSWLcttbY+bp54AQVPqWdMSAggggAACCCCAAAIIIIAAAggggAACCCCAAAIIIOAoAYImR5WTwSCAAAIIIIAAAggggAACCCCAAAIIIIAAAggggAACqSdA0JR61rSEAAIIIIAAAggggAACCCCAAAIIIIAAAggggAACCDhKgKDJUeVkMAgggAACCCCAAAIIIIAAAggggAACCCCAAAIIIIBA6gkQNKWeNS0hgAACCCCAAAIIIIAAAggggAACCCCAAAIIIIAAAo4SIGhyVDkZDAIIIIAAAggggAACCCCAAAIIIIAAAggggAACCCCQegIETalnTUsIIIAAAggggAACCCCAAAIIIIAAAggggAACCCCAgKMECJocVU4GgwACCCCAAAIIIIAAAggggAACCCCAAAIIIIAAAgikngBBU+pZ0xICCCCAAAIIIIAAAggggAACCCCAAAIIIIAAAggg4CgBgiZHlZPBIIAAAggggAACCCCAAAIIIIAAAggggAACCCCAAAKpJ0DQlHrWtIQAAggggAACCCCAAAIIIIAAAggggAACCCCAAAIIOErABE2XJPk6alQMBgEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBA4HYLRJmg6VdJRW53S9wfAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEDAUQK7vE6fiRjp5eX1vKOGxWAQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQRuq0BsbOwor9PnzlXwitH629oSN0cAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEHCUQKy3HvQyI2JWk6PqymAQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAgdsqYGYzZQoKbG+DJnOcOXv+aym21m1tlZsjgAACCCCAAAIIIIAAAggggAACCCCAAAIIIIAAAgikcQGvJUEZ/WubQcQHTeY3zGxK43Wl+wgggAACCCCAAAIIIIAAAggggAACCCCAAAIIIIDAbRSIm8kU10SioMmGTefOVVB0bBsvL6+qkgpK8r2N/eHWCCCAAAIIIIAAAggggAACCCCAAAIIIIAAAggggAAC7isQJemP2NjYVfLxGp8pIGBDwq7+DwExdrBCbhYLAAAAAElFTkSuQmCC
```

Comment:
The fixes to the sizing logic made this control image being incorrect visible...we _we're_ clipping the bottom off.